docs/UserGuide.md

```diff
@@ -50,77 +49,85 @@ data_coNdUctorS is a **desktop address book application for managing contact det
+//& Found why the clicking of Features didnt work, issue I forgot, but the fix is in _markbind/layouts/default.md
+
 //& CHECK IF REPLACING PARAMETERS WITH FIELDS MAKE SENSE <br>
 //& MIGHT WANT TO KEEP PARAMETERS, esp. on the part on any order. IF SO, TO UPDATE PICTURE
 
 <box type="info" seamless>
 
-### General Notes about command format
+### General notes about Command Format
 
 * Terminologies used to refer to a specific part of the command format are shown below:
 ![Description for Command Line](images/commandLineDescription.png)
 
-* Words in `UPPER_CASE` are the **fields** ~~parameters~~ to be supplied by the user.<br>
-  e.g. in `add n/NAME`, `NAME` is a **field** ~~parameter~~ which can be used as `add n/John Doe`.
+* Words in `UPPER_CASE` are the **fields** (underlined in image) to be supplied by the user.<br>
+  e.g. in `add n/NAME`, `NAME` is a **field** which can be used as `add n/John Doe`.
 
-* Items in square brackets are optional.<br>
+* Items in square brackets are **optional**.<br>
   e.g `n/NAME [nn/NICKNAME]` can be used as `n/John Doe nn/johnnyboiiii` or as `n/John Doe`.
 
 * Items with `…`​ after them can be used from one to multiple times.<br>
-  e.g. `[r/ROLE]…​` can be used twice as `r/External r/Marketing` etc.
+  e.g. `r/ROLE…​` can be used twice as `r/External r/Marketing` etc.
 
 * Parameters can be in any order.<br>
   e.g. if the command specifies `n/NAME e/EMAIL`, `e/EMAIL n/NAME` is also acceptable.
 
 * Extraneous parameters **//& OR FIELDS? BUT SOUND WEIRD** for commands that do not take in parameters (such as `exit` and `clear`) will be ignored.<br>
   e.g. if the command specifies `exit 123`, it will be interpreted as `exit`.<br>
 //& ARE HELP AND LIST CONSIDERED "COMMANDS THAT DO NOT TAKE IN PARAMETERS?"
+// no
 
-* The COMMAND_WORD is case-insensitive. (eg. add; ADD; aDd are all interpreted as ADD FEATURE)
+* The COMMAND_WORD is **case-insensitive**. (eg. add; ADD; aDd are all interpreted as ADD FEATURE)
 
 * A limit of 1000 characters //& REPHRASE IF NEEDED
 
 * If you are using a PDF version of this document, be careful when copying and pasting commands that span multiple lines as space characters surrounding line-breaks may be omitted when copied over to the application.
   </box>
 
+
+<div style="page-break-after: always;"></div>
+
 ### Adding a contact: `add`
+Format:<br>`add n/NAME th/TELEGRAM_HANDLE e/EMAIL s/STUDENT_STATUS r/ROLE…​ [nn/NICKNAME]`
 
-Adds a contact to the address book. (labels do not need to be in order, NICKNAME is optional)
-//& LABELS OR PARAMETERS?
+<box type="definition" seamless>
 
-Format:
-* `add n/NAME th/TELEGRAM_HANDLE e/EMAIL s/STUDENT_STATUS r/ROLE…​ [nn/NICKNAME]`
+**Action:** Adds a contact to the address book. (parameters do not need to be in order, NICKNAME is optional)
+</box>
 
 <box type="tip" seamless>
 
-**Tip:** A contact can have one to many roles which match one of the following
+**Tip:** A contact must have at least one role which matches\* the following:
```

Comment:
This suggests that only one of the roles need to match the role list

Reply:
**Tip:** A contact must have at least one role. Roles should match\* the following: